### PR TITLE
Update verbiage to work w/ LearnIDE & local

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,18 @@
...
+2. While this is marked as a Readme, you should code-along with this document.
+   If you have access to a Flatiron School sandbox (you'll see a blue button up
+   top), launch it. If you're working on a local system, simply open a new
+   terminal window.
+3. Once you're in a terminal environment, issue `git clone <clone source
+   path>`. You'll find the path by clicking on the "Clone or download" button
+   at the [repo][].
+4. `cd` into the cloned directory e.g. `cd mechanics-of-migrations-v-000/`
+5. Ensure you can `bundle` in the directory: `bundle install`
+6. If that works, then you're ready to proceed.
```
Comment (gj) [inline README.md]: I can imagine students saying, "And what if it didn't work?!?!"

**Review by gj — COMMENTED** [review]
LGTM! Left one comment about an error branch in the logic b/c I don't have extremely high hopes about how well our current curriculum / help docs prepare students to take the initiative to read error messages / debug common issues on their own. In my experience, there's been a *ton* of handholding required

File: README.md (hunk 1 above)
Comment (sgharms) [reply README.md]: I assume that's what AAQ is for....

Comment (sgharms) [reply README.md]: Amended, hopefully in a generic way.

Comment (gj) [reply README.md]: Yeah haha, you're def right about that — most of that burden falls upon AAQ. Guess it really comes down to more of a product call on how that burden should be split between excruciatingly exhaustive curriculum (more effort to produce / update) and greater reliance on AAQ (which would eventually probably necessitate more staffing)

**Pull request merged**